index.html

```diff
@@ -3,12 +3,48 @@
   <head>
     <meta charset="UTF-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.13.1/css/all.min.css"
+    />
+
     <link rel="stylesheet" href="/styles/style.css" />
     <title>ThemeMascot Covid-19 UI</title>
   </head>
   <body>
     <nav id="nav" class="nav">
-      <div class="container-nav"></div>
+      <div class="container-nav">
+        <div class="brand">
+          <a href="index.html">
+            <img
+              src="./assets/logo-white.png"
+              alt="ThemeMascot"
+              class="brand-img"
+            />
+          </a>
+        </div>
+        <input type="checkbox" class="toggler" id="toggler" />
+        <div class="menu-wrap">
+          <ul class="menu-links">
+            <li><a class="menu-link" href="#home">Home</a></li>
+            <li><a class="menu-link" href="#about">About</a></li>
+            <li><a class="menu-link" href="#symptoms">Symptoms</a></li>
+            <li>
+              <a class="menu-link" href="#prevention">Prevention</a>
+            </li>
+            <li><a class="menu-link" href="#doctors">Doctors</a></li>
+            <li><a class="menu-link" href="#blog">Blog</a></li>
+            <li>
+              <a href="#" class="btn btn-link"
+                ><i class="fa fa-mobile"></i> +347697804</a
+              >
+            </li>
+          </ul>
+        </div>
+        <label for="toggler" class="toggler-label">
+          <span></span>
+        </label>
+      </div>
```

Comment:
Also can you please pay attention to the little details in replicating the page? When on a smaller screen and the hamburger menu is visible, it's not properly vertically aligned. 

<img width="738" alt="Screenshot 2020-07-10 at 2 02 39 PM" src="https://user-images.githubusercontent.com/8373327/87157496-4541d480-c2b6-11ea-8ccd-5ce16501fbae.png">

It's also missing a thin underline from the main website.
I understand you are adding the blue background to the nav cause the other section isn't built out yet but i'd prefer you add that background to whatever parent is holding the nav (Create one if it doesn't exist)

Reply:
Will correct on navigation branch